pom.client.xml

```diff
@@ -646,5 +646,6 @@
     <module>./core</module>
     <module>./keyvault/client</module>
     <module>./tracing</module>
+    <module>./storage/client</module>
```

Comment:
I'm not sure if US alphabets are different, but storage should be before tracing alphabetically where I'm from :-)